src/main/java/mods/railcraft/common/util/misc/MiscTools.java

```diff
@@ -50,10 +50,6 @@ public static void registerTrack(EnumTrack rail) {
             }
     }
 
-    public static String cleanTag(String tag) {
-        return tag.replaceAll("[Rr]ailcraft\\p{Punct}", "").replaceFirst("^tile\\.", "").replaceFirst("^item\\.", "");
```

Comment:
Yeah the registry name stuff is on my todo list, but the tag cleaning is used everywhere from localization to config files.